test/runtests.jl

```diff
@@ -7,3 +7,89 @@ for prob in names(OptimizationProblems)
   prob_fn = eval(prob)
   prob_fn()
 end
+
+# Compare against CUTEst
+import CUTEst
+const problems_cutest = CUTEst.select()
+
+using NLPModels, JuMP, MathProgBase
+
+path = joinpath(dirname(@__DIR__), "src")
+files = filter(readdir(path)) do filename
+    endswith(filename, ".jl") && filename ∉ ("OptimizationProblems.jl", "template.jl")
+end
+
+struct Status
+    samenvars::Bool
+    samexval::Bool
+    sameobjval::Bool
+    samegradval::Bool
+end
+
+cutest_problem(name) = CUTEst.CUTEstModel(name)
+function optpr_problem(name, args...)
+    fmodel = getfield(OptimizationProblems, Symbol(name))
+    omodel = fmodel(args...)
+    x0 = getvalue.(omodel[:x])
+    d = JuMP.NLPEvaluator(omodel)
+    MathProgBase.initialize(d, [:Grad])
+    x0 = getvalue.(omodel[:x])
```

Comment:
We created `NLPModelsJuMP` for this.

Reply:
Interestingly, I get an `OutOfMemoryError()` with `NLPModelsJuMP` but not with `JuMP.NLPEvaluator`.